### ISSUE TITLE
Weird loss curve

### ISSUE THREAD

**Issue by Andrew-Zhang** [Yxxxb/VoCo-LLaMA#11]
Hello! I followed the instructions to train VoCO-LLaMA, and I get the following loss curve.

![image](https://github.com/Yxxxb/VoCo-LLaMA/assets/43453481/66423150-82ca-4ba5-9e79-340d69792492)

Is it normal to have spikes where the loss drops to around 0.3 roughly every 20 steps? Thanks!

**Comment by Yxxxb** [comment]
Hi!

I encountered a similar situation when training LLaVA, which seems to be a common phenomenon in large (visual) language models. This is not caused by VoCo-LLaMA.
[This](https://www.fast.ai/posts/2023-09-04-learning-jumps/) is a possible reasonable explanation I have learned.
Reducing the learning rate or using LoRA may alleviate this phenomenon.

**Comment by Andrew-Zhang** [comment]
Ok thanks!
